llvm/lib/Target/WebAssembly/WebAssemblyCFGStackify.cpp

```diff
@@ -1767,10 +1764,7 @@ bool WebAssemblyCFGStackify::runOnMachineFunction(MachineFunction &MF) {
 
   // Add an end instruction at the end of the function body.
   const auto &TII = *MF.getSubtarget<WebAssemblySubtarget>().getInstrInfo();
-  if (!MF.getSubtarget<WebAssemblySubtarget>()
-           .getTargetTriple()
-           .isOSBinFormatELF())
```

Comment:
This condition is a relic of the era that we didn't have our own wasm binary format and can be removed.